tests/FixedArrayTest.php

```diff
@@ -166,6 +166,64 @@ public function providerShiftDown() : array
         ];
     }
 
+    public function testCreate()
+    {
+        $fixedArray = FixedArray::fromArray([1, 2, 3, 4]);
+        $result = $fixedArray->create();
+
+        $this->assertSame(0, $result->getSize());
+        $this->assertInstanceOf(FixedArray::class, $result);
+    }
+
+    public function testToArray()
+    {
+        $expectedArray = [1, 2, 3, 4];
+        $fixedArray = FixedArray::fromArray($expectedArray);
+        $result = $fixedArray->toArray();
+
+        $this->assertCount(4, $result);
+        foreach($result as $key => $value) {
+            $this->assertSame($expectedArray[$key], $value);
+        }
```

Comment:
You can replace these 4 lines with `$this->assertSame($expectedArray, $result);`